litgpt/api.py

```diff
@@ -302,30 +302,12 @@ def distribute(
         if generate_strategy in ("sequential", "tensor_parallel") and accelerator not in ("cuda", "gpu"):
             raise NotImplementedError(f"generate_strategy='{generate_strategy}' is only supported for accelerator='cuda'|'gpu'.")
 
-        if devices == "auto":
-            if generate_strategy in ("sequential", "tensor_parallel"):
-                total_devices = CUDAAccelerator.auto_device_count()
-            else:
-                total_devices = 1
-        elif isinstance(devices, int):
-            use_devices = calculate_number_of_devices(devices)
-            total_devices = CUDAAccelerator.auto_device_count()
-            if use_devices > total_devices:
-                raise ValueError(
-                    f"You selected more devices ({use_devices}) than available in your system ({total_devices})."
-                )
-            else:
-                total_devices = use_devices
+        num_devices = calculate_number_of_devices(devices)
```

Comment:
Any reason why this was changed? Bad rebase maybe?

Reply:
Yeah. Not sure when it happened.